packages/plugins/plugin-deck/src/components/DeckLayout/ComplementarySidebar.tsx

```diff
@@ -71,7 +71,6 @@ export const ComplementarySidebar = ({ panels, current }: ComplementarySidebarPr
                 <Surface
                   key={id}
                   role={`complementary--${panel}`}
-                  limit={1}
```

Comment:
How do we handle precedence? Currently if we set the limit Sheet’s settings don’t render because Space’s generic rename field gets filled first.

Could we set a chain of fallbacks perhaps?

Reply:
Surfaces can have a disposition, currently the object settings is specified as fallback https://github.com/dxos/dxos/blob/0077e1e08f073c9a329e72c7c5d15baf9c6d25d3/packages/plugins/plugin-space/src/SpacePlugin.tsx#L455 so any others should be rendered first. If that's not the case perhaps some other plugin is currently being too greedy and not specific enough with what it's rendering for

Reply:
I’d creeped scope to handle this, so I’ll revert back and leave this to be solved separately.